README.md

```diff
@@ -137,6 +155,12 @@ It is possible to both implement copy-on-write `ArrayBuffer`s and keep the "fixe
 
 At this time, Google Chrome deems this mitigation important enough for security to not implement CoW `ArrayBuffer`s.
 
+### How `transfer` and `get detached()` is used in browsers and in runtimes?
```

Comment:
This seems more accurate as just "How is `get detached()` used in browsers and in runtime?"

Reply:
I'll open another pull request to include the references for `transfer`